tests/common/dualtor/dual_tor_utils.py

```diff
@@ -1132,9 +1132,15 @@ def delete_neighbor(duthost, neighbor):
         duthost.shell("ip neighbor del %s dev %s" % (neighbor, neighbor_details['dev']))
     else:
         logging.info("Neighbor entry %s doesn't exist", neighbor)
+        return
 
     neighbor_details = get_neighbor(duthost, neighbor)
-    assert len(neighbor_details) > 0, "server ip {} hasn't been deleted from neighbor table.".format(neighbor)
+    # Try the deletion again if the first time failed.
```

Comment:
This scenario pretty much fit into the wait_until, delay with some retry and assert when it failed to delete.

Reply:
@StormLiangMS The reason I add one more try is the entry can't be deleted in rare cases, so I add second try if it failed. It doesn't have to wait time_out and check if neighbor exists time by time. Delete, check, delete it again, otherwise the case should be failed.